src/strings/needleman_wunsch/helpers.rs

```diff
@@ -0,0 +1,311 @@
+//! Helper functions for the Needleman-Wunsch algorithm.
+
+use crate::number::UInt;
+
+/// The direction of best alignment at a given position in the DP table
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum Dir {
```

Comment:
I don’t love Dir as a name, because I’m so used to it meaning Directory rather than Direction. What about “From” or “Where”?

Reply:
This is in a private module so the user doesn't see it. Morgan had named it `Direction` but I shortened it just so have a smaller looking table in the test at line 265. I can change it back before we merge.

Reply:
Done